app/cdap/components/ConnectionList/Components/TabLabelCanSample/index.tsx

```diff
@@ -26,22 +26,28 @@ import { createRef, Ref, useContext, useEffect, useState } from 'react';
 import { Redirect } from 'react-router';
 import { getCurrentNamespace } from 'services/NamespaceStore';
 import useStyles from './styles';
+import { useLocation } from 'react-router';
+import T from 'i18n-react';
 
 export default function TabLabelCanSample({
   label,
   entity,
   initialConnectionId,
   toggleLoader,
   setIsErrorOnNoWorkSpace,
+  testid,
```

Comment:
the prop can be passed around as `dataTestId`, but on the element themselves, please use `data-testid={dataTestId}` as Sean mentioned below.

Reply:
Done